gdnative-core/src/core_types/dictionary.rs

```diff
@@ -286,9 +289,11 @@ impl<Access: NonUniqueThreadAccess> Dictionary<Access> {
     ///
     /// It isn't thread-safe to perform operations that change the container
     /// size from multiple threads at the same time.
-    /// Creating multiple `Unique` references to the same collections, or
+    /// Creating multiple [`Unique`] references to the same collections, or
     /// violating the thread-safety guidelines in non-Rust code will cause
     /// undefined behavior.
+    ///
+    /// [`Unique`]: ../../prelude/struct.Unique.html
```

Comment:
This is `gdnative-core` which does not have a `prelude`. Only `gdnative` has, but this will not work on `docs.rs` I think?

Reply:
Oh, I see. It is difficult to build these links so that both the public and private docs are correct. I'm not sure what to do in this case. I'll open a different PR and split out the error corrections.